src/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer.c

```diff
@@ -25,14 +26,43 @@ ActorProfile Npc_Engineer_Profile = {
 };
 #endif
 
-#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer/aNEG_actor_ct.s")
+//#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer/aNEG_actor_ct.s")
```

Comment:
Could you remove the GLOBAL_ASM lines now that the functions are matched please?